onert-micro/luci-interpreter/src/kernels/Mul.cpp

```diff
@@ -180,6 +180,7 @@ void execute_kernel_CircleMul(const circle::Operator *cur_op, BaseRuntimeGraph *
     }
     break;
 #ifndef DIS_QUANT
+      // TODO: check quantize Mul
```

Comment:
results were incorrect, need to find the issue and then can add tests for quantize version